test.sh

```diff
@@ -26,7 +26,6 @@ function test {
   go test $(go list ./... | grep -v /vendor/)
 }
 
-test "sqlite"
```

Comment:
Please add this back, SQLite is still supported by pop, just off by default. Also, removing this isn't related to this PR. Thanks.

Reply:
Got it.  I misunderstood.  I've added this line back in.